es8/2017-03/mar-21.md

```diff
@@ -0,0 +1,1457 @@
+
+MM: I don't think anything we've discussed affects whether we should proceed to stage 3.
+
+DH: The point of order is that I would like us to have a plan to address this.
+
+MM: I think we're agreed on that
+
+AWB: I think it's clear that today that Annex B is the place we have to put it. There's an orthogonal issue of: what do we want to do with Annex B (whatever's in that bucket). ??
+
+DH: I don't know what the criteria are for Annex B
+
+BE/WH: Let's have the discussion at the proper time.
+
+BE: We can assume a good outcome and if something goes bad we can discussion
+
+MM: Dynamically scoped is also spooky action-at-a-distance. Any objections to stage 3?
+
+BE: I'm so sorry
+
+DE: There's one point I want to make about it. The subclasses restrictions. There are some regex classes in the wild. XRegExp is an example, which is an old-style subclass instance `of RegExp`, so it doesn't really get touched by the detection in this proposal which doesn't modify `new target`, but it seems a little bit odd... The justification for restricting it from subclasses is that it would give bad answers for specific subclasses for example… it might leak implementation details. XRegExp will get past that because it won't get detected.
+
+AWB: Well it doesn't not get detected, it just doesn't meet the definition of what we call a subclass.
+
+DE: Well, but this document mentions certain concerns which XRegExp should qualify for. And it could leak implementation details in these dollar sign things (special variables)... maybe it could cause unexpected behavior.
+
+MM: I think the trade-off is highly in favor of doing with Claude codifying (preventing of sub-classing of spooky...). Are you objecting on this basis?
+
+DE: No, I'm not objecting on this basis.
+
+WH: Who were the reviewers for this?
+
+DE: I was a reviewer
+
+AK: then it cannot go to stage 3 until we have another reviewer.
+
+WH: There are things in there that I don't understand. Like why it has anything to do with inheritance.
+
+MM: I'm happy to clarify with you offline.
+
+MF: we also agreed that any proposal planning to advance would be available 1 or 2 weeks (I forget which) ahead of the meeting.
+
+AK: it's nice to have it advance if it could. But what we got out of it is that it needs eyes on it.
+
+MM: withdrawn from stage 3 for now.  
+
+AWB: are you planning to present it at the next meeting?
+
+MM: yes.
+
+DH: Add it to the agenda for next
```

Comment:
I very much agree with this. This is especially true for numeric computing use cases where scripting is done in JavaScript. Having to reach for the entire WASM toolchain would be __very__ burdensome.